go.mod

```diff
@@ -17,6 +17,7 @@ require (
 	github.com/apparentlymart/go-cidr v1.1.0
 	github.com/aws/aws-sdk-go v1.38.46 // indirect
 	github.com/axw/gocov v1.0.0
+	github.com/azure/azure-sdk-for-go v56.2.0+incompatible // indirect
```

Comment:
It semes like conflicts were resolved incorrectly. Try to run `go mod tidy` and `go mod vendor`: it removes changes from `go.mod`, `go.sum` and `vendor/modules.tx`.